README.md

```diff
@@ -18,7 +18,7 @@
 | Sublimination   | [MikaelSchultz](https://community.home-assistant.io/u/MikaelSchultz) posted [Sublimination Theme](https://community.home-assistant.io/t/sublimination-theme/67312)   | sublimination       |
 | Black and Green | [GreenTurtwig](https://community.home-assistant.io/u/GreenTurtwig) posted [Black and Green Theme](https://community.home-assistant.io/t/black-and-green-theme/28602) | teal                |
 | Vintage         | [surendran.anup](https://community.home-assistant.io/u/surendran.anup) posted [Vintage Theme](https://community.home-assistant.io/t/vintage-theme/42806)             | vintage             |
-| Amoled          | [lance36] posted [Amoled Theme](https://community.home-assistant.io/t/amoled-theme/94875)                                                                            | amoled             |
+| Amoled          | [lance36](https://community.home-assistant.io/u/lance36) posted [Amoled Theme](https://community.home-assistant.io/t/amoled-theme/94875)                                                                            | amoled             |
```

Comment:
Could you please put your theme at the top of the list?
It's alphabetical ordered (based on variable name).